### PR TITLE
CUJ Tweaks for Setting up my developer account

### LINKED ISSUE #1037: [CWS CUJs] I can provide all the information needed to create my developer account.
[Friction Log](https://docs.google.com/document/d/1V_XoWbhYneFiDWu-WEHBUdHDJqwo0ISI_4T4jHYaZEg/edit?usp=sharing)

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #1037
CC @CharyCh 

Changes proposed in this pull request:

- See [Friction Log](https://docs.google.com/document/d/1V_XoWbhYneFiDWu-WEHBUdHDJqwo0ISI_4T4jHYaZEg/edit?usp=sharing) Action Steps

**Staged Preview links**

- [Create and Setup a developer account](https://deploy-preview-1080--developer-chrome-com.netlify.app/docs/webstore/publish/#create-setup-a-developer-account)
- [Certify your data use practices](https://deploy-preview-1080--developer-chrome-com.netlify.app/docs/webstore/cws-dashboard-privacy/#certify-your-data-use-practices)

### CODE REVIEW

**Review by samthor — APPROVED** [review]
Needs a merge but look fine.

File: site/en/docs/webstore/publish/index.md [hunk 1]
```diff
@@ -53,18 +53,57 @@ the ZIP file and manifest depend on the specifics of your item.
...
-Before you can publish on the Chrome Web Store, you need to [register as a Chrome Web Store
-developer][10]. We suggest using a new account just for your item instead of your personal account.
-
-If you already host your item in Google Play and you want your Chrome Web Store listing to show an
-"Available for Android" link, your item must have the same name as your Google Play listing, and
-both items must be owned by the same developer account. To transfer your Chrome Web Store item to a
-different developer, submit this [form][11].
+## Create and setup a developer account {: #create-setup-a-developer-account }
+
+Before you can publish your item on the Chrome Web Store, you'll be asked to pick a developer account to own your items and setup your developer account.
```
Comment (awfuchs) [inline site/en/docs/webstore/publish/index.md:58]: A few issues here:

- "you'll be asked" uses both [future tense](https://developers.google.com/style/tense) (will) and [passive voice](https://developers.google.com/style/voice) (be asked), both of which we avoid where possible.
-  "a developer account to own your items and setup your..." seemed confusing.

My suggestion:

"...Store, you need to pick a developer account to own your items, then set up your account in the Chrome Web Store.

File: site/en/docs/webstore/publish/index.md [hunk 2]
```diff
@@ -53,18 +53,57 @@ the ZIP file and manifest depend on the specifics of your item.
...
+  <thead>
+    <tr>
+      <th>Field</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>Publisher name</strong></td>
+      <td>This name appears under the items' title. If you are a <a href="/docs/webstore/cws-dashboard-listing/#displaying-your-verified-publisher-status">verified publisher</a>, you can display an official publisher URL instead.
```
Comment (awfuchs) [inline site/en/docs/webstore/publish/index.md:84]: In "under the items' title" it's unclear whether this is one item or multiple. (It implies a single title for multiple items.) Perhaps "under the title of each of your items" would be clearer?

File: site/en/docs/webstore/publish/index.md [hunk 3]
```diff
@@ -53,18 +53,57 @@ the ZIP file and manifest depend on the specifics of your item.
...
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/lBcTQm6QF1FBzOmLEfG1.png", alt="Chrome Web Store Account page", width="800", height="404" %}
+
+Here you can provide your developer profile information, configure management settings and enable email notifications. However, only the name, email and privacy policy link are mandatory.
+
+<table>
+  <thead>
+    <tr>
+      <th>Field</th>
+      <th>Description</th>
+    </tr>
```
Comment (awfuchs) [inline site/en/docs/webstore/publish/index.md:79]: Can we do this table in Markdown?

**Review by awfuchs — CHANGES_REQUESTED** [review]
Looks good but I have a few comments I'd like to see addressed.

File: site/en/docs/webstore/publish/index.md (hunk 3 above)
Comment (AmySteam) [reply site/en/docs/webstore/publish/index.md:79]: Of course!

File: site/en/docs/webstore/publish/index.md (hunk 2 above)
Comment (AmySteam) [reply site/en/docs/webstore/publish/index.md:84]: Done!

File: site/en/docs/webstore/publish/index.md (hunk 1 above)
Comment (AmySteam) [reply site/en/docs/webstore/publish/index.md:58]: Got it!

**Review by awfuchs — APPROVED** [review]

**Pull request merged**